src/models/player.rs

```diff
@@ -114,6 +115,8 @@ pub struct Player {
     pub paused: Option<bool>,
     #[serde(skip_serializing)]
     pub seek_history: Vec<SeekLog>,
+    #[serde(skip_serializing)]
+    pub skip_gaps: Option<(SkipGapsRequest, Loadable<SkipGapsResponse, CtxError>)>,
```

Comment:
Maybe instead of exposing the raw api response, we should have some explicit structure for it and core would just make a list of skip chapters like `SkipChapter { start, end, type/name = Enum { INTRO, OUTRO } }` 
As now each client will somehow need to decide which values to use from the response, where probably core has all the data needed to decide consistently across clients which gaps need to be returned to the client.

Reply:
This will be exposed in the subsequent PR. This currently only fetches the skip gaps from the API.

Reply:
That's also the reason why it is skipped when serialising